Funcky.Analyzer/Funcky.Analyzer.CodeFixes/Funcky.Analyzer.CodeFixes.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
```

Comment:
What's the reason behind having the analyzers and the code fixes in separate projects? Wouldn't it be easier if they were in one project?

Reply:
Thats how the template sets the structure up, Roslyn best practices.